recipes/openmpi/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set version = "2.0.0" %}
+{% set major = ".".join(version.split(".")[:2]) %}
+
+package:
+    name: openmpi
+    version: {{ version }}
+
+source:
+    fn: openmpi-{{ version }}.tar.bz2
+    url: https://www.open-mpi.org/software/ompi/v{{ major }}/downloads/openmpi-{{ version }}.tar.bz2
+    sha1: a34f07a0e94af4ccd8080d74827befd61824c6c3
+
+build:
+    number: 0
+    skip: True  # [win]
+    binary_relocation: true
+    detect_binary_files_with_prefix: true
+
+requirements:
+    build:
+        - perl
+        - gcc  # [linux]
+    run:
+        - libgcc  # [linux]
+
+test:
+    files:
+        - tests/helloworld.c
+        - tests/helloworld.cxx
+    commands:
+        - command -v ompi_info
```

Comment:
Adding the `command` fixes the issue we are seeing in https://travis-ci.org/conda-forge/staged-recipes/builds/155618904


```shell
+mpicc --help

Cannot open configuration file /Users/travis/miniconda/envs/_build_placehold_placehold_placehold_placehold_plac/share/openmpi/mpicc-wrapper-data.txt

Error parsing data file mpicc: Not found
```